pandas/tests/scalar/timedelta/test_timedelta.py

```diff
@@ -309,8 +309,13 @@ def test_iso_conversion(self):
         assert to_timedelta('P0DT0H0M1S') == expected
 
     def test_nat_converters(self):
```

Comment:
can you parameterize over box

Reply:
Do you mean over the input to `to_timedelta`?  That would marginally reduce code, whereas parametrizing over `box` would mean completely re-writing this test and probably others.

Reply:
for box, you are testing box=False, need to have equivalent for box=True (which we probably already have), but nicer to have in one place.

Reply:
After looking at this a little more, I don't think parametrizing over box makes sense; the code for the relevant assertions isn't shareable.  Makes more sense to collect the related tests to be adjacent in an upcoming test-refactoring pass.

Reply:
ok fair